### DISCUSSION TITLE
64-bit CopyQ Build

### DISCUSSION THREAD

**Discussion by Kitty131** [hluk/CopyQ#1782] [General]
All - 

Is there a 64-bit build of CopyQ?  Please forgive me - I am not sophisticated at figuring this out.  I see two downloads that, in part, read "amd64", but I don't think that means those are 64-bit builds (?).  In case it matters, all my machines are intel and running 64-bit Windows.  Thank you for your help.

**Comment by hluk** [comment]
The provided Windows builds in [releases](https://github.com/hluk/CopyQ/releases) are 64-bit. They are built by [Appveyor](https://ci.appveyor.com/project/hluk/copyq/history).

**Reply by Shakil-Shahadat** [reply]
Is it possible to add '64bit' text with the windows setup file name? That would remove any further confusion.

**Reply by hluk** [reply]
I would rather avoid that. Most people don't care nor understand it.

**Comment by Kitty131** [comment]
Thank you, hluk!

**Comment by Kitty131** [comment]
The weird thing is that it installs in the x86 folder, where only 32-bit
programs usually install.
C:\Program Files (x86)

However, Windows Process Explorer and Task Manager seem to recognize it as
a  64-bit program.  This was what caused my confusion.

Thank you.

Waqar

**Reply by Shakil-Shahadat** [reply]
I have already reported this https://github.com/hluk/CopyQ/issues/1792.
